### PR TITLE
feat: add configuration page

### LINKED ISSUE #4: Add functionality to the Connect button on the /projects page
The `/projects` page is showing a list of all the user's projects. Those that are Shape Up projects already show a button saying "Go to project" but the rest have a button saying "Connect". The latter is doing nothing as of now. We should either show an explanation of how to configure your project or try to configure the project ourselves automatically.

Captura de pantalla 2023-06-09 a las 18 18 39

### PR DESCRIPTION
**Pull request by fmvilas**
Fixes #4 

Also, refactors a bunch of stuff making use of nested layouts.

### CODE REVIEW

File: src/components/Bet.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function Bet({
     <div className={`flex shadow-sm rounded-md ${className}`}>
       <div className="flex-1 flex items-center justify-between border border-gray-200 bg-white rounded-r-md truncate">
         <div className="flex-1 px-4 py-2 text-sm leading-5 truncate">
-          <a href={issue.url} target="_blank" className="text-gray-900 font-medium hover:text-gray-600 transition ease-in-out duration-150" title={issue.title}>{issue.title}</a>
+          <a href={issue.url} target="_blank" className="text-gray-900 font-medium hover:text-gray-600 transition ease-in-out duration-150" title={issue.title} rel="noreferrer">{issue.title}</a>
```
Comment (Amzani) [inline src/components/Bet.js:14]: @fmvilas curious to know why we don't want referrer links

**Review by Amzani — APPROVED** [review]
Big LGTM :)

**Comment by Amzani** [comment]
@fmvilas once I finish the configuration I have nothing, is it same for you ?

Screenshot 2023-06-30 at 09 06 07

File: src/components/Bet.js (hunk 1 above)
Comment (fmvilas) [reply src/components/Bet.js:14]: Eslint was complaining that this is a security issue not to have it. See https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

**Comment by fmvilas** [comment]
🤔 Interesting. It should redirect you to the Cycle view. I wonder why it's not working.

**Comment by Amzani** [comment]
@fmvilas I've added a fix for #20

**Comment by Amzani** [comment]
Fixed the redirect issue after project configuration

**Pull request merged**